src/RestEase/QueryAttribute.cs

```diff
@@ -8,15 +8,25 @@ namespace RestEase
     [AttributeUsage(AttributeTargets.Parameter | AttributeTargets.Property, Inherited = false, AllowMultiple = false)]
     public sealed class QueryAttribute : Attribute
     {
+        private string _name;
+
         /// <summary>
-        /// Gets or sets the name of the query param. Will use the parameter / property name if null
+        /// Gets or sets the name of the query param. Will NOT use the parameter / property name if null.
```

Comment:
I think it'd be better to say "Will use the parameter / property name if unset"?